volume/provisioners/jiva/snapshot_create.go

```diff
@@ -19,27 +19,19 @@ import (
 // Snapshot will create the snapshot of a given volume name 'volname' with name of
 // snapshot 'snapname'. If there is no name provided for snapshot, an auto genrated
 // string will be genrated for this.
-func Snapshot(volname string, snapname string, labels map[string]string) (string, error) {
+func Snapshot(volname string, snapname string, controllerIP string, labels map[string]string) (command.SnapshotOutput, error) {
```

Comment:
Too many parameters!

Reply:
actually i don't need volname here, i will remove this.